inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/metrics/GCMetricsSysTest.java

```diff
@@ -19,6 +19,8 @@
 
     private static final ViewManager viewManager = Stats.getViewManager();
 
+    public static List blackhole;
```

Comment:
What is the purpose of storing the trash in a variable?

Reply:
When using a loop like this

    for (int i = 0; i < 1000000; i++) {
        new ArrayList<>();
    }
        
the compiler notices that the constructed object is not assigned and not used at all, thus, it will (most likly) "optimize" it away. So, during the test no garbage will be produced anymore.
Having a variable like this, the compile cannot know if this variable is and will be used because basically, it could/can be used by any class due to its `public` modifier - thus it cannot be safely removed by the optimizer.